patient-service/src/main/java/com/raf/si/patientservice/service/impl/SchedMedExaminationServiceImpl.java

```diff
@@ -137,10 +137,10 @@ public SchedMedExamResponse deleteSchedMedExamination(Long id) {
                     log.info(errMessage);
                     throw new BadRequestException(errMessage);
                 });
-
-
+        Patient patient= scheduledMedExamination.getPatient();
+        scheduledMedExamination.setPatient(null);
         scheduledMedExamRepository.delete(scheduledMedExamination);
-
+        scheduledMedExamination.setPatient(patient);
```

Comment:
Ovo nema smisla, zasto bi ponovo setovao pacijenta? Prosledi pacijenta kao parametar funkcije

Reply:
Tako sam uradio da ne bi morao duplitari komandu ( ako bih pisao novu ), a morao sam da set na null da ne bi izbrsiao pacijenta. Tbh ne bi trebalo da utice na back to je cisto da bi vratio frontu i podatke o pacijentu. 
Ukoliko bi promenio, morao bih  da menjam svuda gde se poziva navedena komanda sto bi  onda bude ovako nest.

```
schedMedExamMapper.schedMedExamToSchedMedExamResponse(schedMedExam, schedMedExam.getPatient())
```

sto mi je tbh , po meni, vise bvz i veci hasstle neg samo da mu ponovo vratim pacijenta

Reply:
Mogu ak os i da im uopste i ne vracam podatke o pacijentu, mislim da im uopste ne trebaju kod brisanja zakazanog pregleda tho trebala bi nam potvrda od fronta oko toga

Reply:
Ne moras da setujes, one to many je veza gde je foreign key samo u pregledu za pacijenta, brisanjem pregleda ne brises pacijenta, mislim da CascadeType po defaultu ne cascade-uje operacije

Reply:
Nisam znao radim na izmeni 👍